charts/quickwit/values.yaml

```diff
@@ -424,10 +447,16 @@ prometheusRule:
   #      severity: warning
 
 service:
+  # Deprecated, please use <name>.service.type
   type: ClusterIP
 
+  # Deprecated, please use <name>.service.annotations
+  # or service.headless.annotations for the headless service
   annotations: {}
```

Comment:
I think it doesn't hurt to keep this to have an annotation that is applied to all services

Reply:
Good point, I don't recall why I wanted to deprecate it :sweat_smile: